deb/postrm

```diff
@@ -25,5 +25,6 @@ if [[ "$1" == "purge" || "$1" == "remove" ]]; then
     if [ $(getent group harvest) ]; then
       groupdel harvest && echo " --> delete harvest group"
     fi
+    rm -rf /opt/harvest
```

Comment:
by default rpm/deb uninstalls will remove files which they installed so harvest.yml will go away. Solution to this would be to backup yml and restore like i did for upgrade cases

Reply:
I think when $1 == "purge", then we should remove everything (maybe ask for confirmation), otherwise keep configs and caches in conventional.

Reply:
Agree. I have made the change to keep harvest.yml and harvest.example.yml during uninstall case. For purge, no questions, everything goes away.
I haven't made any changes w.r.t conf folder as part of this story. Let's discuss on how do we handle /conf folder seperately.